From f7f4bdd020dafd60f870d3857fe3aea09fc4cabf Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Fri, 24 Mar 2017 10:37:46 +0800
Subject: [PATCH] arm: dts: rk3288-evb: disable uart 1/3/4

Uart3 has been iomux to gpio, for vcc_3g regulator, which is designed
on rk3288 evb main board.

Disable unused uarts to fix gpio request blaim during system booting.

Change-Id: I2eb79ae63a6f226255c12fc3da9ba95ec4219d32
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-evb.dtsi | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/arch/arm/boot/dts/rk3288-evb.dtsi b/arch/arm/boot/dts/rk3288-evb.dtsi
index 778020305861..30fe23251f3b 100644
--- a/arch/arm/boot/dts/rk3288-evb.dtsi
+++ b/arch/arm/boot/dts/rk3288-evb.dtsi
@@ -470,22 +470,10 @@
 	status = "okay";
 };
 
-&uart1 {
-	status = "okay";
-};
-
 &uart2 {
 	status = "okay";
 };
 
-&uart3 {
-	status = "okay";
-};
-
-&uart4 {
-	status = "okay";
-};
-
 &vpu_service {
 	status = "okay";
 };
-- 
2.35.3

